Core/Program.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
What's the change here?

Reply:
No idea, but it does absolutely nothing. I never touched that file.

Reply:
If it's an unknown change it shouldn't be commited. Before commiting you should check all changes and commit only the intended ones.

Actually it seems that `using System;` is not even needed here, so you might want to try removing it completely.